opt_adapt/opt.py

```diff
@@ -600,13 +587,7 @@ def minimise(
 
         if it > 2 and mesh_adaptation:
             J_ = op.J_progress[-2]
-            if nc < nc_:
-                mesh_adaptation = False
-                adaptor = identity_mesh
-                if params.disp > 1:
-                    pprint("NOTE: turning adaptation off due to mesh converged")
-                continue
```

Comment:
What made you decide to drop this?

Reply:
Because I found that if we choose a large default target like 3000, this condition may not suitable.